xSharePoint.sln

```diff
@@ -0,0 +1,39 @@
+﻿
```

Comment:
Can you remove the SLN file here? We have moved on from using Visual Studio 2015 to Visual Studio code